tools/ci_build/github/azure-pipelines/orttraining-linux-gpu-frontend-test-ci-pipeline.yml

```diff
@@ -1,42 +1,257 @@
+    condition: succeededOrFailed() # ensure all tests are run
+
+  - script: |
+      docker run \
+        --gpus all \
+        --shm-size=1024m \
+        --rm \
+        --volume $(Build.SourcesDirectory):/onnxruntime_src \
+        --volume $(Build.BinariesDirectory):/build \
+        --volume /bert_data:/bert_data \
+        onnxruntime-ubuntu16.04-cuda10.1-cudnn7.6 \
+          /build/RelWithDebInfo/launch_test.py \
+            --process_args "python orttraining_run_glue.py ORTGlueTest.test_roberta_fp16_with_mrpc -v" \
+            --cwd /build/RelWithDebInfo \
+            --env "CUDA_VISIBLE_DEVICES 2"
+    displayName: 'Run orttraining_run_glue.py ORTGlueTest.test_roberta_fp16_with_mrpc'
+    condition: succeededOrFailed() # ensure all tests are run
+
+  - script: |
+      docker run \
+        --gpus all \
+        --shm-size=1024m \
+        --rm \
+        --volume $(Build.SourcesDirectory):/onnxruntime_src \
+        --volume $(Build.BinariesDirectory):/build \
+        --volume /bert_data:/bert_data \
+        onnxruntime-ubuntu16.04-cuda10.1-cudnn7.6 \
+          /build/RelWithDebInfo/launch_test.py \
+            --process_args "python orttraining_run_multiple_choice.py ORTMultipleChoiceTest.test_bert_fp16_with_swag -v" \
+            --cwd /build/RelWithDebInfo \
+            --env "CUDA_VISIBLE_DEVICES 2"
+    displayName: 'Run orttraining_run_multiple_choice.py ORTMultipleChoiceTest.test_bert_fp16_with_swag'
+    condition: succeededOrFailed() # ensure all tests are run
+
+  - script: |
+      docker run \
+        --gpus all \
+        --shm-size=1024m \
+        --rm \
+        --volume $(Build.SourcesDirectory):/onnxruntime_src \
+        --volume $(Build.BinariesDirectory):/build \
+        --volume /bert_data:/bert_data \
+        onnxruntime-ubuntu16.04-cuda10.1-cudnn7.6 \
+          /build/RelWithDebInfo/launch_test.py \
+            --process_args "python onnxruntime_test_ort_trainer_with_mixed_precision.py -v" \
+            --cwd /build/RelWithDebInfo
+    displayName: 'Run onnxruntime_test_ort_trainer_with_mixed_precision.py'
+    condition: succeededOrFailed() # ensure all tests are run
+
+  - script: |
+      docker run \
+        --gpus all \
+        --rm \
+        --volume $(Build.SourcesDirectory):/onnxruntime_src \
+        --volume $(Build.BinariesDirectory):/build \
+        --volume /bert_data:/bert_data \
+        onnxruntime-ubuntu16.04-cuda10.1-cudnn7.6 \
+          /build/RelWithDebInfo/launch_test.py \
+            --process_args "python orttraining_test_transformers.py BertModelTest.test_for_pretraining_mixed_precision -v" \
+            --cwd /build/RelWithDebInfo
+    displayName: 'Run orttraining_test_transformers.py BertModelTest.test_for_pretraining_mixed_precision'
+    condition: succeededOrFailed() # ensure all tests are run
+
+  # I have tried following to ways to run run_training_pipeline_e2e_tests. both failed.
```

Comment:
>run_training_pipeline_e2e_tests [](start = 42, length = 31)

What's the plan for pipelin_e2d_tests? Do you plan to move it together with run_training_python_frontend_e2e_tests and run_training_python_frontend_tests?

Reply:
I am still working on it, however I tried 2 methods and both has failed. There are other issues with this PR. I may come to you for help once I am to fix it again.

Reply:
adding following code in teh script seemed to work. 
if __name__ == "__main__":
    sys.exit(main())